### PR TITLE
RAC-5606 CoreOS maximum OS install payload

### PR DESCRIPTION
**Pull request by gpaulos**
One more for CoreOS

@jimturnquist @larry-dean

### CODE REVIEW

**Comment by jimturnquist** [comment]
Test this please

File: test/config/install_coreos_maximum_QuantaT41.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+    "bootstrap-payload": {
+        "name": "Graph.InstallCoreOS",
+        "options": {
+            "defaults": {
+                "hostname": "rackhdnode",
```
Comment (hohene) [inline test/config/install_coreos_maximum_QuantaT41.json]: all your other max payloads have a user of 'rackhd-node'.  Wasn't sure if you're tying to follow a pattern here?

Comment (jimturnquist) [inline test/config/install_coreos_maximum_QuantaT41.json]: @hohene Did you get an answer on the above from George?  I noticed you approved the changes.

Comment (gpaulos) [inline test/config/install_coreos_maximum_QuantaT41.json]: CoreOS does not allow dashes in the username.

**Pull request merged**